# Uses/python.py (concurrent) doesnt find
# share/* stuff for concurrent processing
# only ^man/man*

--- setup.cfg.orig	2019-12-27 05:05:29 UTC
+++ setup.cfg
@@ -42,7 +42,7 @@ backends = 
 	upt-openbsd
 
 [options.data_files]
-share/man/man1 = 
+man/man1 = 
 	upt.1
 
 [egg_info]
